R/qview.R

```diff
@@ -106,6 +125,9 @@ queueView <- function(
   # Initialize our return values, which we will use later to feed results from the
   #   module into the calling Shiny App.
   .return_values <- shiny::reactiveValues()
+  shiny::observe(
+    .return_values$rows <- .rv$queue_data
+  )
 
   # Create an observe event that re-runs (via invalidation) every 1000 milliseconds.
```

Comment:
is the invalidation still at 1 second intervals, or now at the .refresh_interval

Reply:
It should be at .refresh_interval, unless you see something to indicate otherwise.

Reply:
Just noticed the comment, I'll clean that up.